test/integration/agent/flowexporter_test.go

```diff
@@ -164,3 +165,17 @@ func TestConnectionStoreAndFlowRecords(t *testing.T) {
 	flowRecords := flowrecords.NewFlowRecords(connStore)
 	testBuildFlowRecords(t, flowRecords, testConns, testConnKeys)
 }
+
+func TestSetupConnTrackParameters(t *testing.T) {
+	err := connections.SetupConntrackParameters()
+	if err != nil {
+		t.Skipf("Skipping test as trying to set up conntrack parameters returned an error: %v", err)
```

Comment:
do we require this gating even after moving this test to the integration test?

Reply:
Thanks for pointing out. I have removed this if condition.